packages/extra/monitoring/templates/vm/vmcluster.yaml

```diff
@@ -6,22 +6,35 @@ metadata:
   name: {{ .name }}
 spec:
   replicationFactor: 2
-  retentionPeriod: "3"
+  retentionPeriod: {{ .retentionPeriod }}
```

Comment:
`| quote` is mandatory here because otherwise someday we will encounter an issue with scientific notation